### ISSUE TITLE
adding top contributors

### ISSUE THREAD

**Issue by sahil-shubham** [metakgp/metakgp.github.io#5]
## Adding top contributors section

Adding the top contributors of the month of metakgp would motivate the open source beginners, attract people who haven't yet contributed to make their first contribution and also mention other people who work hard to improve the projects.

### How to proceed
- Use the [ Github API](https://docs.github.com/en/free-pro-team@latest/rest) to get the list of top contributors of the month (based on commits) from each repository. 
- Then combine all the above lists to get to get the overall top contributors for the organization.
- Display the above top contributors with their name and github profile link.

**Comment by sahil-shubham** [comment]
Hey @thealphadollar, I would like to work on this. I haven't worked with Github API, which means I would be learning a lot.

**Comment by thealphadollar** [comment]
Yeah, that would be surely a good learning experience. Interacting with APIs is fun too :P and you are going to have to solve a lot of "bugs" :stuck_out_tongue:

**Comment by sahil-shubham** [comment]
@thealphadollar I forgot about taking this issue, apologies for the delay. I would create a PR in a few days, and would keep further discussions there.

**Comment by thealphadollar** [comment]
Thanks for remembering, Sahil.
